applications/asset_tracker_v2/src/modules/Kconfig.cloud_module

```diff
@@ -32,12 +32,28 @@ choice NRF_CLOUD_SEND_SOCKET
 	default NRF_CLOUD_SEND_TIMEOUT
 endchoice
 
-# MQTT Transport library
-config MQTT_CLEAN_SESSION
-	default y
+choice NRF_CLOUD_CLIENT_ID_SRC
+	default NRF_CLOUD_CLIENT_ID_SRC_RUNTIME
+endchoice
+
+if BOARD_QEMU_X86
+
+config NRF_CLOUD_FOTA
```

Comment:
Why is it needed to redefine this symbol at application level?

Reply:
You're right. This change is not necessary. I updated the `prj_qemu_x86.conf` to explicitly disable `NRF_FOTA`.